src/energy_comms/transform/epa.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+
+
+def transform(df):
+
+    # from HUD: https://www.huduser.gov/portal/datasets/usps_crosswalk.html
+
+    zip_code_crosswalk = pd.read_excel(
+        "/Users/mcastillo/Documents/GitHub/rmi-energy-communities/src/energy_comms/extract/ZIP_COUNTY_122021.xlsx",
+        dtype={"zip": str, "county": str},
+    )
+
+    zip_crosswalk = dict(zip(zip_code_crosswalk["zip"], zip_code_crosswalk["county"]))
+
+    df = df.rename(columns={"Site Name": "site_name", "Zip Code": "zip"})
```

Comment:
I'd suggest not renaming zip code because zip code is more descriptive than just "zip" and it's not necessary to rename it for the `pd.map` call.